src/core/fonts.js

```diff
@@ -3353,8 +3353,10 @@ function type1FontGlyphMapping(properties, builtInEncoding, glyphNames) {
         charCodeToGlyphId[charCode] = 0; // notdef
       }
     }
-  } else if (isSymbolicFont) {
+  } else if (isSymbolicFont || properties.hasCustomBuiltInEncoding) {
```

Comment:
Did you notice that the fonts are actually CIDFonts (but their embedded data streams are basic Type1 fonts (non-CIDFonts)). CIDFonts should not be affected by Encodings, so I think your solution based on  `hasCustomBuiltInEncoding` is not the correct one for CIDFonts. Instead I suggest the following fix

```diff
diff --git a/src/core/fonts.js b/src/core/fonts.js
index 614048c9..761bd39c 100644
--- a/src/core/fonts.js
+++ b/src/core/fonts.js
@@ -3599,6 +3599,19 @@ var Type1Font = (function Type1FontClosure() {

     getGlyphMapping: function Type1Font_getGlyphMapping(properties) {
       var charstrings = this.charstrings;
+
+      if (properties.composite) {
+        const charCodeToGlyphId = Object.create(null);
+        let glyphId, charCode;
+        // Map CIDs directly to GIDs.
+        for (glyphId = 0; glyphId < charstrings.length; glyphId++) {
+          charCode = properties.cMap.charCodeOf(glyphId);
+          // Add 1 because glyph 0 is duplicated.
+          charCodeToGlyphId[charCode] = glyphId + 1;
+        }
+        return charCodeToGlyphId;
+      }
+
       var glyphNames = [".notdef"],
         glyphId;
       for (glyphId = 0; glyphId < charstrings.length; glyphId++) {
```

Thus each CID is mapped to an equal GID value. A similar CID mapping can be found in `CFFFont.prototype.getGlyphMapping`.

Reply:
> Did you notice that the fonts are actually CIDFonts (but their embedded data streams are basic Type1 fonts (non-CIDFonts)).

I'm aware of that :-)
The reason for the current approach was that it, at the time, seemed better to trust the *actual* font program rather than the font dictionary (since we've seen time and time again how the latter can be wrong in all kind of "interesting" ways, which thus breaks font rendering).

However your suggestion does seem more in-line with the specification, so assuming that it fixes the issue and doesn't cause regressions elsewhere, it might indeed be better.
I'll update the PR once I've had time to run all tests; thank you!

Reply:
That seems to have worked; thanks again!